package.json

```diff
@@ -9,22 +9,25 @@
     "backend-deploy": "aws s3 cp src/backend/transcriber/backend.zip s3://media-analysis-demo/build/ --acl public-read",
     "build": "npm run webui-build && npm run lambda-build && npm run cfn-build && npm run backend-build",
     "cfn-build": "npm run cfn-package && npm run lambda-inline",
-    "cfn-deploy": "aws s3 cp ./templates/template.yaml s3://media-analysis-demo/ --acl public-read",
+    "cfn-deploy": "npm run zip && aws s3 cp ./v$npm_package_version.zip s3://$CFN_BUCKET/$npm_package_name/v$npm_package_version/$npm_package_name.zip",
```

Comment:
How do these environment variables work? I don't see them defined in the in the `.travis.yml` file?

Reply:
when running npm scripts all `package.json` fields are available as vars prefixed with `npm_package_` - https://docs.npmjs.com/misc/scripts#packagejson-vars

`CFN_BUCKET` is defined on the repo in Travis as it's not a public bucket. although I could add it in Travis encrypted via `travis encrypt`, since it is only available meant to be available on master it seemed best to be in the repo settings according to https://docs.travis-ci.com/user/environment-variables/. what do you think?

Reply:
Ah if that's how they recommend it then let's leave it.